src/Traits/HasWallet.php

```diff
@@ -227,4 +227,18 @@ public function transfers(): HasMany
             ->hasMany(config('wallet.transfer.model', Transfer::class), 'from_id')
         ;
     }
+
+    /**
+     * returns all the receiving transfers to this wallet.
+     *
+     * @return HasMany<Transfer>
+     */
+    public function payments(): HasMany
```

Comment:
1. This should be in the Wallet model;
2. The name is not good, because this is the receipt of a transfer (there are transactions: transfer, exchange, payment);
3. No unit tests;

Reply:
Ok, what should this relationship be called?

Reply:
You can call it like this:
```php
receivedTransfers
```
I don't have a better idea.

---

According to the unit test, it is not necessary to write a new one. It is enough to pull a new method in an existing test and check the number of elements.

Reply:
And, most importantly. If you want a feature now, then PR must be done from the 10.x branch.

If from the master, then it is already 11.x, and this version will not support laravel 10.x